docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM pytorch/pytorch:1.6.0-cuda10.1-cudnn7-runtime
+
+
+RUN apt-get update && apt-get upgrade -y
```

Comment:
In general, we need to have as small number of `RUN` statements as possible due to image size. The single `RUN` statement increases the size of image. So, please do as follows:
```
RUN apt-get update && \
         apt-get upgrade -y && \
         apt-get install -y gcc libxml2-dev libxslt1-dev zlib1g-dev g++ git cmake && \
         pip install --upgrade pip && \
         pip install Cython && \
         pip install d3rlpy git+https://github.com/takuseno/d4rl-pybullet git+https://github.com/takuseno/d4rl-atari
```